pages/common/rar.md

```diff
@@ -0,0 +1,27 @@
+# rar
+
+> An archive manager with compression ratio.
+
+- Archive file/files:
+
+`rar a {{archive_name.rar}} {{path/to/file1}} {{path/to/file2}} {{path/to/file3}}`
+
+- Archive a directory:
+
+`rar a {{archive_name.rar}} {{path/to/directory}}`
+
+- Archive and slit the file/files into parts of eqaul size (50M):
+
+`rar a -v50M -R {{archive_name.rar}} {{path/to/file_or_directory}}`
+
+- Archive with password protected:
+
+`rar a -p{{password}} {{archive_name.rar}} {{path/to/file_or_directory}}`
+
+- Archive with password protected where even file lists are encrypted:
+
+`rar a -h{{password}} {{archive_name.rar}} {{path/to/file_or_directory}}`
```

Comment:
I tested this example and didn't work, I checked the man page of the command and found the switch -hp, if you are referring to this switch, I would go with following description and example:

```
Archive, ecrypting file data and headers with password:
 
rar a -hp{{password}} {{archive_name.rar}} {{path/to/file_or_directory}}
```